service_configuration_lib/spark_config.py

```diff
@@ -935,7 +937,6 @@ def _get_k8s_spark_env(
         'spark.kubernetes.pyspark.pythonVersion': '3',
         'spark.kubernetes.container.image': docker_img,
         'spark.kubernetes.namespace': 'paasta-spark',
-        'spark.kubernetes.container.image.pullPolicy': 'Always',
```

Comment:
this is unnecessary and will waste time/data since we're telling k8s to never use the cached version of the image. since we always include the full tag, there's no reason to skip the cache since we never re-use tags:)